tests/certification/state/cockroachdb/cockroachdb_test.go

```diff
@@ -0,0 +1,281 @@
+		if err != nil {
+			panic(err)
+		}
+		defer client.Close()
+
+		err = stateStore.Ping()
+		assert.Equal(t, nil, err)
+
+		resp, err := stateStore.Get(&state.GetRequest{
+			Key: keyOneString,
+		})
+		assert.NoError(t, err)
+		assert.Equal(t, "2", *resp.ETag) // 2 is returned since the previous etag value of "1" was incremented by 1 when the update occurred
+		assert.Equal(t, "\"Overwrite Success\"", string(resp.Data))
+
+		return nil
+	}
+
+	//ETag test
+	eTagTest := func(ctx flow.Context) error {
+		etag1 := "1"
+		etag100 := "100"
+
+		// Setting with nil etag will insert an item with an etag value of 1 unless there is a conflict
+		err := stateStore.Set(&state.SetRequest{
+			Key:   keyOneString,
+			Value: "v1",
+		})
+		assert.NoError(t, err)
+
+		// Setting with an etag wil do an update, not an insert so an error is expected since the etag of 100 is not present
+		err = stateStore.Set(&state.SetRequest{
+			Key:   keyOneString,
+			Value: "v3",
+			ETag:  &etag100,
+		})
+		assert.Equal(t, fmt.Errorf("no item was updated"), err)
+
+		resp, err := stateStore.Get(&state.GetRequest{
+			Key: keyOneString,
+		})
+		assert.NoError(t, err)
+		assert.Equal(t, etag1, *resp.ETag)           //1 is returned since the default when the new data is written is a value of 1
+		assert.Equal(t, "\"v1\"", string(resp.Data)) //v1 is returned since it was the only item successfully inserted with the key of keyOneString
+
+		// This will update the value stored in key K with "Overwrite Success" since the previously created etag has a value of 1
+		// It will also increment the etag stored by a value of 1
+		err = stateStore.Set(&state.SetRequest{
+			Key:   keyOneString,
+			Value: "Overwrite Success",
+			ETag:  &etag1,
+		})
+		assert.NoError(t, err)
+
+		resp, err = stateStore.Get(&state.GetRequest{
+			Key: keyOneString,
+		})
+		assert.NoError(t, err)
+		assert.Equal(t, "2", *resp.ETag) //2 is returned since the previous etag value of "1" was incremented by 1 when the update occurred
+		assert.Equal(t, "\"Overwrite Success\"", string(resp.Data))
+
+		return nil
+	}
+
+	// Transaction related test - also for Multi
+	transactionsTest := func(ctx flow.Context) error {
+
+		// Set state to allow for a delete operation inside the multi list
+		err = stateStore.Set(&state.SetRequest{Key: certificationTestPrefix + "key1", Value: []byte("certificationdata")})
+		assert.NoError(t, err)
+
+		// get state
+		item, errUpdatedGet := stateStore.Get(&state.GetRequest{Key: certificationTestPrefix + "key1"})
+		assert.NoError(t, errUpdatedGet)
+		assert.Equal(t, []byte("certificationdata"), item.Data)
+
+		err = stateStore.Multi(&state.TransactionalStateRequest{
+			Operations: []state.TransactionalStateOperation{
+				{
+					Operation: state.Upsert,
+					Request: state.SetRequest{
+						Key:   "reqKey1",
+						Value: "reqVal1",
+						Metadata: map[string]string{
+							"ttlInSeconds": "-1",
```

Comment:
This component doesn't have TTL support and this test doesn't assert anything related to TTL. Can you please remove the TTL metadata here throughout?